### ISSUE TITLE
permisstion CNI installation dir

### ISSUE THREAD

**Issue by kobayashi** [weaveworks/ignite#585]
While installing CNI binary with [install docs](https://ignite.readthedocs.io/en/stable/installation.html#cni-plugins), some commands were failed because of permission to the dest dir.
The part should be like this.

```
export CNI_VERSION=v0.8.5
export ARCH=$([ $(uname -m) = "x86_64" ] && echo amd64 || echo arm64)
sudo mkdir -p /opt/cni/bin
curl -sSL https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-linux-${ARCH}-${CNI_VERSION}.tgz | sudo tar -xz -C /opt/cni/bin
```

**Comment by stealthybox** [comment]
Fixed by #586

**Comment by stealthybox** [comment]
Thank you for fixing this!
I've had to edit these commands many times.
